libraries/postgres/src/Version.js

```diff
@@ -4,6 +4,7 @@ const Method = require('./Method');
 const { loadSql } = require('./util');
 
 const objMap = (obj, fn) => Object.fromEntries(Object.entries(obj).map(fn));
+const ALLOWED_KEYS = ['version', 'migrationScript', 'downgradeScript', 'methods'];
```

Comment:
argh, you always catch my issues before I get to r- you

Reply:
Python's `set` takes an iterator as an arg to its constructor, so I always forget that the JS `Set` doesn't..